docs/_docs/integration-checks-reference.md

```diff
@@ -718,13 +718,15 @@ To resize an image or convert an image to PNG, you can use this
 
 <a name="M005"></a><a name="M00005"></a>
 
-## M005 - Developer Email Domain Matches App Domain
+## M005 - Admin Email Domain Matches App Domain
 
-To ensure that this integration is being submitted by the app owner we require that
-one of the developers listed on the project have an email address with the same
+To ensure that this integration is being submitted by the app owner, we require that
+one of the admins listed on the project have an email address with the same
```

Comment:
I'd suggest updating this line's wording to [reflect the UI](https://cdn.zappy.app/bc5ed27b5e5894fc9971f27b3c106a79.png), so instead of _project_, maybe _Manage Team page_? Feel free to disregard if you think this would require a more considerable overarching change, and I'll accept the review after that.